pkg/controller/operators/catalog/operator_test.go

```diff
@@ -142,38 +143,75 @@ func TestTransitionInstallPlan(t *testing.T) {
 
 func TestSyncInstallPlanUnhappy(t *testing.T) {
 	namespace := "ns"
+	ipWithSteps := withSteps(installPlan("p", namespace, v1alpha1.InstallPlanPhaseInstalling, "csv"),
+		[]*v1alpha1.Step{
+			{
+				Resource: v1alpha1.StepResource{
+					CatalogSource:          "catalog",
+					CatalogSourceNamespace: namespace,
+					Group:                  "",
+					Version:                "v1",
+					Kind:                   "ServiceAccount",
+					Name:                   "sa",
+					Manifest: toManifest(t, serviceAccount("sa", namespace, "",
+						objectReference("init secret"))),
+				},
+				Status: v1alpha1.StepStatusUnknown,
+			},
+		},
+	)
 
 	tests := []struct {
-		testName string
-		in       *v1alpha1.InstallPlan
-		err      error
+		testName       string
+		in             *v1alpha1.InstallPlan
+		err            error
+		expectedPhase  v1alpha1.InstallPlanPhase
+		operatorGroups []*operatorsv1.OperatorGroup
+		clientObjs     []runtime.Object
```

Comment:
I think `*OperatorGroup` implements `runtime.Object`. Can these fields be merged?

Reply:
Done